ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,74 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        machine_config_selector (object): Matching labels/expressions, to determine which MachineConfig objects
+            to apply this MachineConfigPool object.
+        configuration (dict): Targeted MachineConfig object for the machine config pool, in the following format:
+            {"name": (str), "source": <List of dicts, each representing a MachineConfig resource>}
+        node_selector_labels (object): Matching labels, to determine to which nodes to apply this
```

Comment:
```suggestion
        node_selector (dict): nodeSelector dict with supported selector logic
```
For example:


  matchLabels:
    component: redis
  matchExpressions:
    - { key: tier, operator: In, values: [cache] }
    - { key: environment, operator: NotIn, values: [dev] }

Reply:
Ack

Reply:
This is not resolved